packages/@aws-cdk/aws-logs-destinations/lib/lambda.ts

```diff
@@ -7,19 +7,20 @@ import { Construct } from '@aws-cdk/core';
  * Use a Lamda Function as the destination for a log subscription
  */
 export class LambdaDestination implements logs.ILogSubscriptionDestination {
-  constructor(private readonly fn: lambda.IFunction) {
+  constructor(private readonly fn: lambda.IFunction, private readonly addPermissions: boolean=true) {
```

Comment:
In all CDK constructs and also integration classes we put these kind of optional configuration arguments in a struct.

That way, when you're specifying it it looks like:

```ts
new LambdaDestination(lambda, { addPermissions: false })
```

Instead of:

```ts
new LambdaDestination(lambda, false)
```

Which is from first glance much easier to see what's going on.

Also, an options struct has the advantage over positional arugments that if we want to add more flags later on, it doesn't end up looking like:

```ts
new LambdaDestination(lambda, false, false, false, true, false, true);
```